MacPass/MPOutlineViewController.m

```diff
@@ -221,7 +221,9 @@ - (void)_didBecomeFirstResponder:(NSNotification *)notification {
     return; // Nothing we need to worry about
   }
   MPDocument *document = self.windowController.document;
-  document.selectedGroups = self.currentTargetGroups;
+  if (document.selectedGroups != self.currentTargetGroups) {
```

Comment:
This fixed the issue of an unexpected exit of search but raises another. When a user clicks inside the outline view and selects the currently selected group this wont show up in the inspector. Instead another group needs to be selected and the the old on should be re-selected. It might be usefull to go a special route just if search is active